expr/functions/aliasByBase64/function.go

```diff
@@ -46,7 +46,8 @@ func (f *aliasByBase64) Do(ctx context.Context, e parser.Expr, from, until int64
 		if withoutFieldArg {
 			decoded, err := base64.StdEncoding.DecodeString(a.Name)
 			if err == nil {
-				r = a.CopyName(string(decoded))
+				r = a.CopyLink()
```

Comment:
This changes saves original tags. But it's really needed after alias ? May be only name tag ?

Reply:
Looks like this change was a mistake on our side, since `CopyName` also updates the `"name"` tag, and that is correct. Will fix.